src/ccl_background.c

```diff
@@ -714,23 +714,39 @@ void ccl_cosmology_distances_from_input(ccl_cosmology * cosmo, int na, double a[
 INPUT: cosmology, scale factor array, growth array, growth rate array
 TASK: if not already there, create growth splines with the input arrays and store them.
 */
-void ccl_cosmology_growth_from_input(ccl_cosmology* cosmo, int na, double a[], double growth_arr[], double fgrowth_arr[], int* status)
+void ccl_cosmology_growth_from_input(ccl_cosmology* cosmo, int na, double a[],
+                                     double growth_arr[], double fgrowth_arr[], int* status)
 {
   int chistatus;
   if (cosmo->computed_growth)
     return;
 
+  double *growth_normed = NULL;
   gsl_spline * growth = gsl_spline_alloc(cosmo->spline_params.A_SPLINE_TYPE, na);
   gsl_spline * fgrowth = gsl_spline_alloc(cosmo->spline_params.A_SPLINE_TYPE, na);
+  double growth0 = growth_arr[na-1];
 
   if (growth == NULL || fgrowth == NULL) {
     *status = CCL_ERROR_MEMORY;
     ccl_cosmology_set_status_message(
         cosmo, "ccl_background.c: ccl_cosmology_growth_from_input(): ran out of memory\n");
   }
 
+  // Need to normalize the growth factor
   if (*status == 0) {
-    chistatus = gsl_spline_init(growth, a, growth_arr, na);
+    growth_normed = malloc(na*sizeof(double));
+    if (growth_normed == NULL) {
+      *status = CCL_ERROR_MEMORY;
+      ccl_cosmology_set_status_message(
+           cosmo, "ccl_background.c: ccl_cosmology_growth_from_input(): ran out of memory\n");
+    }
+  }
+
+  if (*status == 0) {
+    for(int ia=0; ia<na; ia++)
+      growth_normed[ia] = growth_arr[ia]/growth0;
+
+    chistatus = gsl_spline_init(growth, a, growth_normed, na);
```

Comment:
We are now storing the normalized growth instead of the unnormalized growth in the same spot. Do we have a test that we are not applying the normalization twice in other places in the code?

Reply:
Tested [here](https://github.com/LSSTDESC/CCL/blob/30138e2bc6c13d01f72c3bb6176d1968c62ade68/pyccl/tests/test_background.py#L157) now